packages/Avatar/README.md

```diff
@@ -9,21 +9,33 @@
 
 ### Usage
 
+By default, Avatar component sets the background and text color using the `getAvatarColor`function with the children props value as the parameter.
+
 ```js
-<Avatar backgroundColor="black" color="white" size="small">
-  A
-</Avatar>
+import Avatar from "@paprika/avatar";
+
+<Avatar>A</Avatar>;
```

Comment:
```suggestion
<Avatar>Terry Fox</Avatar>;
```

Reply:
For now, `<Avatar>` is setup to take the values pass on  `children` prop (string or icon). For example ```<Avatar>Terry Fox</Avatar>``` 
![avatar](https://user-images.githubusercontent.com/67071062/86033012-1fcae600-b9ed-11ea-83db-d8664c81465f.png)
Since it is brought up, I guess it makes sense (and improves DX) to define how to display the `children` prop values. If we are going to approach the `Avatar` like the above example. Are we going to set the initial to _uppercase_ and _one letter only_ (first character)?

Reply:
Oh, interesting 🤔 

If the `typeof children === 'string'` it's possible we could re-use the logic from the `acl-ui` `<Avatar>`:
https://github.com/acl-services/acl-ui/blob/master/src/components/__beta__/Avatar/Avatar.js#L10-L31

But if it's a `node` (Icon)` I guess we'd have to just return that. This is different than the way the `acl-ui` version works though.